src/components/dialogs/dataSources/COVIDcast.svelte

```diff
@@ -12,7 +13,7 @@
   let signal = '';
   let geo_type = '';
   let geo_value = '';
-  let form_key = '';
+  let form_key = $apiKey;
```

Comment:
do we need `form_key` here, or can we get rid of it the same way we did with the `auth` variable in the other `.../dataSources/*.svelte` files?